### ISSUE TITLE
Sometimes not fetching work when using CUDA; CPUs go idle

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#783]
**Reported by Nicolas on 13 Nov 38979084 07:33 UTC**
I'm creating a new ticket because the other two were about the scheduler, and this looks like a work fetch issue.

```
[<zombie67> i have an 8-way with two cores sitting idle
[23:49](23:49]) <zombie67> it refuses to ask for work, from the one project with work available
[<zombie67> oops, now 4 cores idle
[23:51](23:50]) <zombie67> It's running 6.4.5...wonder if upgrading to 6.5 would help
[<PovAddict> zombie67: I think 6.5 lacks some last-week (not to say last-minute) fixes that were done in 6.4 branch...
[23:53](23:53]) <PovAddict> so I think 6.5 would be worse in that area
[<zombie67> I just upgraded to 6.5
[23:54](23:54]) <zombie67> I will give it some time to think about getting work
[<zombie67> anyone successfully downgraded from 6.4 or 6.5 to 6.2?
[23:56](23:55]) <PovAddict> I never went to 6.4
[<zombie67> down to 6 idle cores now
[23:56](23:56]) <zombie67> ugh
[<zombie67> 6.2.18 was the last of the windows 6.2?
[23:57](23:56]) <PovAddict> .19
[00:06] <zombie67> ding!  6.2.19 is asking for work as it ought to
```

Reported on `#boinc`, `irc.freenode.net`.

**Comment by romw** [comment]
**Commented by zombie67 on 23 Oct 38979109 02:40 UTC**
This is on XP64, using he XP64 client.  It would not ask for more tasks to be downloaded, even though it was down to two running, with 6 idle cores.

**Comment by romw** [comment]
**Commented by zombie67 on 16 Jul 38979119 08:26 UTC**
It happened again on a second machine.  Downgrading to 6.2.19 fixes the problem.

**Comment by romw** [comment]
**Commented by romw on 17 Nov 39180706 05:46 UTC**
Fixed in 6.6
